R/dbFetch.R

```diff
@@ -75,7 +99,10 @@ NULL
     rv[[chunk.count]] <- chunk
     chunk.count <- chunk.count + 1
   }
-  if (length(rv) == 1) {
+  if (length(rv) == 0) {
```

Comment:
I'd prefer if we didn't do this. If the server does not return anything even when we have fetched all chunks, then it's either a server bug or we are not handling errors properly at some other spot.